ecosystem/gstreamer_plugin/gst_mtl_common.c

```diff
@@ -287,6 +287,12 @@ void gst_mtl_common_init_general_arguments(GObjectClass* gobject_class) {
       g_param_spec_boolean("enable-ptp", "Enable onboard PTP",
                            "Enable onboard PTP client", FALSE,
                            G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property(
+      gobject_class, PROP_GENERAL_LCORE_MASK,
+      g_param_spec_string("lcore-mask", "dpdk core mask",
```

Comment:
naming this "mask" suggests that it should be used like  `--lcore-mask 00101`, but it should be used like `--lcore-list 2,4`. Consider renaming it.